doc/docs/additional.md

```diff
@@ -1,91 +1,73 @@
 # Additional topics
 
 ## Authentication
+
 Any openIDConnect provider can be used to supply oAuth credentials.
 
 The required set of parameters are:
 
-* oidConnectUrl
-* clientId
-* clientSecret
-* oAuthUsername
-* oAuthPassword
+- oidConnectUrl
+- clientId
+- clientSecret
+- oAuthUsername
+- oAuthPassword
 
 You can add oAuth authentication to the pipeline by providing the required set
-of parameters via the command line or in
-the `/pipelines/controller/config/application.yaml` file.
+of parameters via the command line or in the
+`/pipelines/controller/config/application.yaml` file.
 
 === "Cmd line"
-    ```java
-    $ java -cp ./pipelines/batch/target/batch-bundled-0.1.0-SNAPSHOT.jar org.openmrs.analytics.FhirEtl \
-        --fhirServerUrl=[FHIR_SERVER_URL] --outputParquetPath=[PATH] \
-        --resourceList=Patient,Encounter,Observation --batchSize=200 \
-        --clientId=[CLIENT_ID] --clientSecret=[CLIENT_SECRET] \
-        --OAuthUsername=[USERNAME] --OAuthPassword=[PASSSWORD] \
-        --oidConnectUrl= [OPENID_CONNECT_URL] \
-    ```
+`java $ java -cp ./pipelines/batch/target/batch-bundled-0.1.0-SNAPSHOT.jar org.openmrs.analytics.FhirEtl \ --fhirServerUrl=[FHIR_SERVER_URL] --outputParquetPath=[PATH] \ --resourceList=Patient,Encounter,Observation --batchSize=200 \ --clientId=[CLIENT_ID] --clientSecret=[CLIENT_SECRET] \ --OAuthUsername=[USERNAME] --OAuthPassword=[PASSSWORD] \ --oidConnectUrl= [OPENID_CONNECT_URL] \ `
 === "application.yaml"
-    ```
-    fhirdata:
-    .....
-        # The following client credentials should be set if the FHIR server accepts
-        # OAuth access tokens. Note the client credentials, e.g., the secret, are
-        # sensitive, and it is probably a better practice to set these through
-        # command-line arguments.
-        fhirServerOAuthTokenEndpoint: "https://path_to_endpoint_for_token"
-        fhirServerOAuthClientId: "THE_CLIENT_ID"
-        fhirServerOAuthClientSecret: "THE_CLIENT_SECRET"
-    ```
+`fhirdata: ..... # The following client credentials should be set if the FHIR server accepts # OAuth access tokens. Note the client credentials, e.g., the secret, are # sensitive, and it is probably a better practice to set these through # command-line arguments. fhirServerOAuthTokenEndpoint: "https://path_to_endpoint_for_token" fhirServerOAuthClientId: "THE_CLIENT_ID" fhirServerOAuthClientSecret: "THE_CLIENT_SECRET"`
```

Comment:
This seems like a bad change; again can we configure spotless not to do changes in code sections?